src/store/DataContext.js

```diff
@@ -0,0 +1,75 @@
+import { createContext, useState, useEffect } from "react";
+
+//creates the context and initial values are set
+const DataContext = createContext({
+    input: "",
```

Comment:
`input` is a local data that only the input field need to be aware, so its good idea to keep in as a local state in the component itself.

Context should only be used for storing data that many distinct components need to access. In this case tasks & toast can be stored in context. Remaining things its better to keep as local states.

Reply:
Resolved